Gemfile

```diff
@@ -4,7 +4,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby '3.1.3'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '6.1.7.5'
+gem 'rails', '6.1.7.6'
```

Comment:
I got confused as to why this was appearing here until I realized the merge target wasn't `development`.